kyo-caliban/src/test/scala/kyo/resolversTest.scala

```diff
@@ -0,0 +1,134 @@
+package kyo
+
+import caliban.*
+import caliban.render
+import caliban.schema.Schema
+import caliban.schema.SchemaDerivation
+import kyo.*
+import kyoTest.KyoTest
+import sttp.model.Uri
+import sttp.tapir.*
+import sttp.tapir.json.zio.*
+import zio.Task
+import zio.ZLayer
+
+class resolversTest extends KyoTest:
+
+    def runZIO[T](v: Task[T]): T =
+        zio.Unsafe.unsafe(implicit u =>
+            zio.Runtime.default.unsafe.run(v).getOrThrow()
+        )
+
+    case class Query(
+        k1: Int < Aborts[Throwable],
+        k2: Int < ZIOs,
+        k3: Int < (Aborts[Throwable] & ZIOs),
+        k4: Int < IOs
```

Comment:
Can you add a field with pending `Fibers` as well?

Reply:
Done 👍